.gitignore

```diff
@@ -22,4 +22,4 @@ todo.md
 # eclipse project files
 .buildpath
 .project
-.settings
+.settings
```

Comment:
Please avoid whitespace changes in files that haven't otherwise changed in order to avoid wrongly updating the "last updated" date.